browser/help/faq/technical-details/what-genetic-ancestry-groups-are-represented-in-the-gnomad-data.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { BaseTable, Link } from '@gnomad/ui'
+
+import { GNOMAD_POPULATION_NAMES } from '@gnomad/dataset-metadata/gnomadPopulations'
+import sampleCounts from '@gnomad/dataset-metadata/sampleCounts'
+
+import TableWrapper from '../../../src/TableWrapper'
+
+const SampleCountTable = styled(BaseTable)`
+  td {
+    text-align: right;
+  }
+
+  tbody tr:last-child {
+    th,
+    td {
+      border-bottom-color: #000;
+    }
+  }
+`
+
+const SampleCountTables = () => (
+  <div>
+    <div>
+      <h4>gnomAD v4</h4>
+      <p>
+        For current stats on genetic ancestry groups present in v4, please see the{' '}
+        {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+        <Link to="/stats">stats page</Link>. For technical details, please see our{' '}
+        {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
+        <Link to="https://gnomad.broadinstitute.org/help/ancestry">Genetic Ancestry in gnomAD</Link>{' '}
+        help text.
+      </p>
+    </div>
+
+    <br />
+    <br />
+    <details>
+      <summary>Expand to see details for past versions</summary>
+      <div>
+        <h4>gnomAD v3</h4>
+        <TableWrapper>
+          <SampleCountTable>
+            <thead>
+              <tr>
+                <th rowSpan={2} scope="col">
+                  Genetic ancestry group
+                </th>
+                <th scope="col">overall</th>
+                <th scope="col">controls/biobanks</th>
+                <th scope="col">non-cancer</th>
+                <th scope="col">non-neuro</th>
+                <th scope="col">non-TOPMed</th>
+                <th scope="col">non-v2</th>
+              </tr>
+            </thead>
+            <tbody>
+              {['afr', 'ami', 'amr', 'asj', 'eas', 'fin', 'mid', 'nfe', 'sas', 'oth'].map(
+                (popId) => (
+                  <tr key={popId}>
+                    {/* @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message */}
+                    <th scope="row">{GNOMAD_POPULATION_NAMES[popId]}</th>
+                    <td>{(sampleCounts.gnomad_r3.genomes[popId] || 0).toLocaleString()}</td>
```

Comment:
I'd rather do this with some proper iteration and so forth but at this point I'm OK with fixing that in post.